doc/user/content/ops/troubleshooting.md

```diff
@@ -109,31 +171,30 @@ number, and anything significantly larger is probably a bug.
 
 ```sql
 -- Extract arrangement records and batches, by worker.
-select mdo.id, mdo.name, mdo.worker_id, mas.records, mas.batches
-from mz_internal.mz_arrangement_sizes as mas,
-     mz_internal.mz_dataflow_operators as mdo
-where
-    mas.operator_id = mdo.id and
+SELECT mdo.id, mdo.name, mdo.worker_id, mas.records, mas.batches
+FROM mz_internal.mz_arrangement_sizes AS mas,
+     mz_internal.mz_dataflow_operators AS mdo
+WHERE
+    mas.operator_id = mdo.id AND
     mas.worker_id = mdo.worker_id
-order by mas.records desc;
+ORDER BY mas.records DESC;
 ```
 
 ```sql
 -- Extract arrangement records and batches, summed across workers.
-select mdo.id, mdo.name, sum(mas.records) as records, sum(mas.batches) as batches
-from mz_internal.mz_arrangement_sizes as mas,
-     mz_internal.mz_dataflow_operators as mdo
-where
-    mas.operator_id = mdo.id and
+SELECT mdo.id, mdo.name, sum(mas.records) AS records, sum(mas.batches) AS batches
+FROM mz_internal.mz_arrangement_sizes AS mas,
+     mz_internal.mz_dataflow_operators AS mdo
+WHERE
+    mas.operator_id = mdo.id AND
     mas.worker_id = mdo.worker_id
-group by mdo.id, mdo.name
-order by sum(mas.records) desc;
+GROUP BY mdo.id, mdo.name
+ORDER BY sum(mas.records) DESC;
 ```
 
 We've also bundled an interactive, web-based memory usage visualization tool to
-aid in debugging. The SQL queries above show all arrangements in Materialize
-(including system arrangements), whereas the memory visualization tool shows
-only user-created arrangements, grouped by dataflow. The amount of memory used
+aid in debugging. The memory visualization tool shows all user-created arrangements,
+grouped by dataflow. The amount of memory used
```

Comment:
What prompted this change?

Reply:
I ran all the queries and did a basic sanity check of the answers vs. the documentation. The queries on `mz_arrangement_sizes` only returned data on user-created arrangements to me, so this prompted the change in the docs. Did I misunderstand something here?

Reply:
I think that actually depends on whether introspection of introspection dataflows is enabled or not. For example:
```
materialize=> create cluster replica default.r3 size '1', introspection debugging true;
CREATE CLUSTER REPLICA
materialize=> set cluster_replica = r3;
SET
materialize=> SELECT mdo.id, mdo.name, sum(mas.records) AS records, sum(mas.batches) AS batches
 FROM mz_internal.mz_arrangement_sizes AS mas,
      mz_internal.mz_dataflow_operators AS mdo
 WHERE
     mas.operator_id = mdo.id AND
     mas.worker_id = mdo.worker_id
 GROUP BY mdo.id, mdo.name
 ORDER BY sum(mas.records) DESC;
 id  |                     name                      | records | batches
-----+-----------------------------------------------+---------+---------
 180 | ArrangeByKey Timely(Histogram)                |   59105 |       4
 142 | ArrangeByKey Timely(Elapsed)                  |   25683 |       4
 294 | ArrangeByKey Timely(MessagesSent)             |   18139 |       5
 332 | ArrangeByKey Timely(MessagesReceived)         |   18130 |       5
 415 | Arrange Timely(Reachability)                  |   10946 |       4
 647 | ArrangeByKey Compute(FrontierCurrent)         |    6336 |       5
 256 | ArrangeByKey Timely(Parks)                    |    5440 |       7
 218 | ArrangeByKey Timely(Addresses)                |     820 |       3
 468 | ArrangeByKey Differential(ArrangementRecords) |     759 |       5
 430 | ArrangeByKey Differential(ArrangementBatches) |     522 |       6
 104 | ArrangeByKey Timely(Channels)                 |     444 |       3
 66  | ArrangeByKey Timely(Operates)                 |     376 |       3
 506 | ArrangeByKey Differential(Sharing)            |      22 |       3
(13 rows)
```

I'm not sure if "system arrangements" exclusively refers to the index-based introspection sources or if there are others. I can't think of any currently. If we add others in the future, those would should up in the SQL query output I guess.

Since the SQL queries currently by default don't show any system arrangements, it's probably a good idea to make the change you suggested, so users are not confused.